tools/test/audio/sof_test_perf_config.m

```diff
@@ -41,6 +41,12 @@
 test.fr_rp_max_db = 1.0;	     % Upper limit for p-p ripple in dB
 test.plot_fr_axis = [10 30e3 -4 1];  % Plot xmin, xmax, ymin, ymax
 
+test.fr_mask_flo = [];
```

Comment:
I would suggest give var more meanings, intead of just test, like for this one: we can use perf_test
for better understanding.